run_migrations.py

```
@@ -57,20 +58,74 @@ def is_first_time():
+            table["name"] if isinstance(table, dict) else table[0]
+            for table in tables
+            if not (table["name"] if isinstance(table, dict) else table[0]).startswith(
+                "aerich"
+            )
+        ]
+
+        return len(user_tables) > 0
+    except (OperationalError, AttributeError) as e:
+        logger.debug("Database check failed: %s", e)
+        return False
+    except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)